.github/workflows/publish-courseware.yaml

```diff
@@ -0,0 +1,18 @@
+name: Publish Docker - learn
+on:
+  push:
+    branches:
+      - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
If insteat of using its own workflow we add this step after the release we can guarnatee we get the right version